tests/functional/conversion/test_encode_structs.py

```diff
@@ -0,0 +1,44 @@
+from ethpm_types.abi import MethodABI
+from hexbytes import HexBytes
+from pydantic import BaseModel
+
+from ape import networks
+
+ABI = MethodABI.parse_obj(
+    {
+        "type": "function",
+        "name": "test",
+        "inputs": [
+            {
+                "name": "s",
+                "type": "tuple",
+                "components": [
+                    {"name": "a", "type": "uint256"},
+                    {"name": "b", "type": "bytes32"},
+                    {"name": "c", "type": "bool"},
+                ],
+            }
+        ],
+    }
+)
+
+
+class Struct(BaseModel):
+    a: int
+    b: bytes
+    c: bool
+
+
+EXPECTED = HexBytes(
+    "0000000000000000000000000000000000000000000000000000000000000001"
+    "3200000000000000000000000000000000000000000000000000000000000000"
+    "0000000000000000000000000000000000000000000000000000000000000001"
+)
+
+
+def test_encode_structs():
+    encode_calldata = networks.ethereum.encode_calldata
+
+    assert EXPECTED == encode_calldata(ABI, (1, b"2", True))
```

Comment:
the "actual" goes first I believe, for some pytest output to make sense.

```suggestion
    assert encode_calldata(ABI, (1, b"2", True)) == EXPECTED
```